### PR TITLE
chore: generate non-cloud client library

### PR DESCRIPTION
**Pull request by JoeWang1127**
Fix https://github.com/googleapis/google-cloud-java/issues/9125

### CODE REVIEW

File: docker/owlbot/java/src/fix-poms.py [hunk 1]
```diff
@@ -274,6 +274,12 @@ def update_bom_pom(filename: str, modules: List[module.Module]):
 
     tree.write(filename, pretty_print=True, xml_declaration=True, encoding="utf-8")
 
+
+def _proto_group_id(group_id: str) -> str:
```
Comment (suztomo) [inline docker/owlbot/java/src/fix-poms.py:278]:
```suggestion
def _proto_group_id(main_artifact_group_id: str) -> str:
```

Comment (suztomo) [inline docker/owlbot/java/src/fix-poms.py:278]: Can you add source code comment why we had to split the proto group IDs?

File: docker/owlbot/java/tests/new-client-maps/golden/grpc-google-maps-foo-v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.maps.api.grpc</groupId>
```
Comment (suztomo) [inline docker/owlbot/java/tests/new-client-maps/golden/grpc-google-maps-foo-v1/pom.xml:5]: Nice test.

**Review by suztomo — APPROVED** [review]
Small suggestion for comment and variable name. Approving this pull request regardless whether you incorporate it or not.

File: docker/owlbot/java/src/fix-poms.py (hunk 1 above)
Comment (JoeWang1127) [reply docker/owlbot/java/src/fix-poms.py]: done

Comment (JoeWang1127) [reply docker/owlbot/java/src/fix-poms.py]: done

**Pull request merged**